src/Symfony/Component/Config/Definition/Builder/ExprBuilder.php

```diff
@@ -76,7 +76,7 @@ public function ifTrue(?\Closure $closure = null): static
      */
     public function ifFalse(?\Closure $closure = null): static
     {
-        $this->ifPart = $closure ?? static fn ($v) => false === $v;
+        $this->ifPart = $closure ? static fn ($v) => false === $closure($v) : static fn ($v) => false === $v;
```

Comment:
I think the strict comparisons should be removed here and in ifTrue also, they just make things more complicated.

Reply:
Given the amount of failing tests, I'm fairly certain some code assumes it to use strict comparisons. 

I'd rather not change all those failing tests... (I locally only ran the tests for `src/Symfony/Component/Config`)